ha/core/system_health/factory/action_factory.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+from ha.core.system_health.handlers.action_handler import NodeActionHandler
+from ha.core.system_health.system_health_exception import InvalidResourceType
+from ha.core.system_health.model.health_event import HealthEvent
+
+EVENT_ACTION_HANDLERS_MAPPING = {
```

Comment:
In this map, we should have only those resource which have corresponding Action Handler implemented?
The NodeActionHandler does not handle failures of individual components within node. If we want to list down all resource types in here, put None in front of those which doesn't have handlers implemented yet and throw exception when these are referenced.

Reply:
Agree, I guess lets mention only those for which corresponding action handler is implemented.

Reply:
Done. Was taken suggestion from Ajay S, to choose the moving forward approach